mkdocs/docs/HPC/multi_job_submission.md

```diff
@@ -22,7 +22,7 @@ huge amounts of small jobs will create a lot of overhead, and can slow
 down the whole cluster. It would be better to bundle those jobs in
 larger sets. In TORQUE, an experimental feature known as "*job arrays*"
 existed to allow the creation of multiple jobs with one *qsub* command,
-but is was not supported by Moab, the current scheduler.
+but is not supported by Moab, the current scheduler.
```

Comment:
I merged this, but we should look into removing all mentions of MOAB as a scheduler, that's ancient history by now (Slurm is both the resource manager and scheduler now, although with a Torque frontend user interface)

Reply:
Referencing #371 for future.